install.ps1

```diff
@@ -63,14 +63,33 @@ param(
     [Switch] $RunAsAdmin
 )
 
+function Write-InstallInfo {
+    param(
+        [Parameter(Mandatory = $True, Position = 0)]
+        $String,
+        [Parameter(Mandatory = $False, Position = 1)]
+        $ForegroundColor
```

Comment:
You could set `[System.ConsoleColor]` so you will get intelisense for colors availble.

```suggestion
		[System.ConsoleColor]
        $ForegroundColor = 'White'
```

Gif:
![A](https://i.imgur.com/4F8kFnr.gif)